File: lazythreetenbp/src/main/java/org/threeten/bp/zone/ZoneRulesCompat.java
package org.threeten.bp.zone;

import static androidx.annotation.RestrictTo.Scope.LIBRARY;

import androidx.annotation.NonNull;
import androidx.annotation.RestrictTo;
import java.io.DataInputStream;
import java.io.IOException;

@RestrictTo(LIBRARY)
public class ZoneRulesCompat {

    @NonNull
    public static ZoneRules readZoneRules(@NonNull DataInputStream stream) throws IOException, ClassNotFoundException {
        return StandardZoneRules.readExternal(stream);
    }

    private ZoneRulesCompat() {
        throw new AssertionError("No instances");
    }
}

File: lazythreetenbp/src/main/kotlin/org/threeten/bp/zone/ZoneRulesCompat.kt
package org.threeten.bp.zone

import androidx.annotation.RestrictTo
import org.threeten.bp.zone.StandardZoneRules
import java.io.DataInputStream
import java.io.IOException
import java.lang.AssertionError

internal object ZoneRulesCompat {

    fun readZoneRules(stream: DataInputStream): ZoneRules =
        StandardZoneRules.readExternal(stream)
}
